packages/app-root/src/components/HomePageContainer.js

```diff
@@ -0,0 +1,24 @@
+'use client'
+
+import { useContext } from 'react'
+import { Box } from 'grommet'
+import { PanoptesAuthContext } from '../contexts'
+
+export default function HomePageContainer({ blogPosts = [] }) {
+  const { isLoading, user } = useContext(PanoptesAuthContext)
+
+  return (
+    <main>
+      {isLoading ? (
+        <Box height='100vh' align='center' justify='center'>
+          Loader goes here
+        </Box>
+      ) : (
+        <Box height='100vh' align='center' justify='center'>
+          <p>{user?.login ? 'Signed-in' : 'Signed-out'}</p>
+        </Box>
+      )}
+      <Box pad='medium' align='center'>Number of blog posts: {blogPosts.length}</Box>
```

Comment:
Not necessary in this PR, but in theory would there be a benefit to this component being a child of the HomePageContainer in page.js? I think it'd then be server side. In `page.js` it'd look something like:
```
<HomePageContainer>
  <Box pad='medium' align='center'>Number of blog posts: {blogPosts.length}</Box>
</HomePageContainer>
```
Then in HomePageContainer it'd be passed along as a child.
I'm not sure there's any benefit, but mentioning just in case.

Reply:
Using any Grommet components means it will be a client component, so I purposely designated HomePageContainer as the entry point into the client component tree.